developer/src/kmc-ldml/src/compiler/compiler.ts

```diff
@@ -67,19 +67,23 @@ export class LdmlKeyboardCompiler {
     const reader = new LDMLKeyboardXMLSourceFileReader(this.options.readerOptions, this.callbacks);
     const data = this.callbacks.loadFile(filename);
     if(!data) {
+      // TODO-LDML: coverage for this
```

Comment:
Do we need coverage TODOs given c8 will have them in its report anyway?

Reply:
We don't but I wanted to distinguish them from false positives. I scanned the coverage report to find these.